src/pages/josungmin/Main/components/Article/Article.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import './Article.scss';
+
+class Article extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      value: '',
```

Comment:
```suggestion
      inputValue: '',
```
value값이면 뭔가 구체적이지 않으니 inputValue값으로 해주시면 더욱 좋지 않을까 싶습니다!

Reply:
"사용자가 댓글 입력한 값" 이라는 의미가 명확히 들어나는 이름을 고민하고 활용해주세요